src/structs/struct-construction.md

```diff
@@ -0,0 +1,43 @@
+# Construction
+
+Unlike in C++ or Java, there is no "constructor" in Rust for structs, all
```

Comment:
I would like to add this content — it's great for the course participants if we can cover constructors.

Please move the file to `methods/constructors.md` instead since constructors are not strictly about structs (they apply to enums too).

Then we just need to fix the small problem pointed out by Andrew and we should be good to go!

Reply:
I would still like to see us cover this. Perhaps as part of @djmitche reorg in #1073.

Reply:
I think this will be in the methods slide?

Reply:
Yeah, I think so too.

Reply:
Yeah, I see it's mentioned on the slide about methods now.